src/subscription.rs

```diff
@@ -156,7 +156,7 @@ impl<T: Debug + Clone + PartialEq, E: Debug + Clone> SubscriptionRegistry<T, E>
                 // subscriptions.lock().unwrap().
                 drop(subscriptions);
 
-                panic!(msg);
+                panic!("{}", msg);
```

Comment:
I'm thinking this might need to be dropped when rebasing.

Reply:
I've just finished the rebase locally and I'm afraid the change remains. Otherwise I get a warning:
```
warning: panic message is not a string literal
   --> src/subscription.rs:160:24
    |
160 |                 panic!(msg);
    |                        ^^^
    |
    = note: `#[warn(non_fmt_panic)]` on by default
    = note: this is no longer accepted in Rust 2021
help: add a "{}" format string to Display the message
    |
160 |                 panic!("{}", msg);
    |                        ^^^^^
help: or use std::panic::panic_any instead
    |
160 |                 std::panic::panic_any(msg);
    |                 ^^^^^^^^^^^^^^^^^^^^^

```